archunit/src/test/java/com/tngtech/archunit/testutil/assertion/JavaAnnotationAssertion.java

```diff
@@ -58,6 +60,39 @@ public JavaAnnotationAssertion hasAnnotationProperty(String propertyName, Annota
         return this;
     }
 
+    public JavaAnnotationAssertion hasExplicitlyDeclaredStringProperty(String propertyName, String expectedValue) {
+        assertThat(actual.hasExplicitlyDeclaredProperty(propertyName))
+                .as("String @%s.%s()" + context() + " has no explicitly declared value", actual.getRawType().getSimpleName(), propertyName)
```

Comment:
This boolean is about whether the property _does_ have an explicitly declared value, isn't it?

Reply:
I've updated my commit on [`add-API-to-retrieve-explicitly-set-annotation-values.review`](https://github.com/TNG/ArchUnit/commits/add-API-to-retrieve-explicitly-set-annotation-values.review). What I forgot to say before: Feel free to squash it, if you like it. 😉

Reply:
Yeah, I should not do some last minute description copy and paste at 2am :disappointed: Thanks a lot for cleaning it up!